.gitlab/container_build/fakeintake.yml

```diff
@@ -4,6 +4,7 @@ docker_build_fakeintake:
   rules:
     - !reference [.except_mergequeue]
     - !reference [.on_fakeintake_changes]
+    - !reference [.on_fakeintake_changes_on_main]
```

Comment:
Is `.on_fakeintake_changes` still needed with this addition?

Reply:
Oh no got it, my bad!

Reply:
Yes, otherwise it won't execute on `main` on changes to `fakeintake`, as the previous rule is "on changes to fakeintake compared to `main`", that is always empty while on `main`